tests/unit/test_cromshellconfig.py

```diff
@@ -0,0 +1,97 @@
+        reload(cromshellconfig)
+        test_cromwell_url = "https://cromwell-v1.dsde-methods.broadinstitute.org"
+
+        assert cromshellconfig.cromwell_server is not None, \
+            "Cromwell server variable should not be empty"
+
+        cromshellconfig.resolve_cromwell_config_server_address(
+            server_user=test_cromwell_url)
+        assert cromshellconfig.cromwell_server == test_cromwell_url, \
+            f"Cromwell server variable should be set to the specified server url : {test_cromwell_url}"
+
+    def test_resolve_cromwell_config_server_address_with_mock_data(self, mock_data_path):
+        reload(cromshellconfig)
+        test_cromwell_url = "https://cromwell-v1.dsde-methods.broadinstitute.org"
+        mock_workflow_id = "d689adec-c600-4e4b-be37-4e30e65848c7"
+        mock_cromwell_url = "https://cromwell-v45.dsde-methods.broadinstitute.org"
+
+        # Set the submission file path to mock data
+        cromshellconfig.submission_file = os.path.join(mock_data_path,
+                                                       "all.workflow.database.tsv")
+        # Execute function that changes cromwell server based off the
+        # workflow id and submission file
+        cromshellconfig.resolve_cromwell_config_server_address(
+            workflow_id=mock_workflow_id)
+        assert cromshellconfig.cromwell_server == mock_cromwell_url, \
+            f"Cromwell server variable should be set to {mock_cromwell_url}"
+
+        # Cromwell url parameter should supersede workflow id look up
+        cromshellconfig.resolve_cromwell_config_server_address(
+            server_user=test_cromwell_url,
+            workflow_id=mock_workflow_id)
+        assert cromshellconfig.cromwell_server == test_cromwell_url, \
+            f"Cromwell server variable should be set to {test_cromwell_url}"
+
+    def test_config_dir(self):
+        reload(cromshellconfig)
+        assert cromshellconfig.config_dir is not None, "Config variable should be set"
+
+        assert cromshellconfig.config_dir == os.path.join(Path.home(), ".cromshell"), \
+            "Config file variable path should be in .cromshell"
+
+        assert Path(cromshellconfig.config_dir).exists(), \
+            "Config directory should exist "
+
+    def test_submission_file(self):
+        reload(cromshellconfig)
+        assert cromshellconfig.submission_file is not None, \
+            "Submission file variable should be set "
+
+        path_to_submission = os.path.join(Path.home(), ".cromshell",
+                                          "all.workflow.database.tsv")
+        assert cromshellconfig.submission_file == path_to_submission, \
+            f"Submission file path should be {path_to_submission} "
+
+        assert Path(cromshellconfig.submission_file).exists(), \
+            "Submission file should exist"
+
+    def test_cromwell_server(self):
+        reload(cromshellconfig)
+        assert cromshellconfig.cromwell_server is not None, \
```

Comment:
There are multiple tests that just tests a file/folder is set (not `None`) and exists, so what about refactoring out an utility method just for that?